modules/tooltip.js

```diff
@@ -0,0 +1,46 @@
+function Tooltip(){}
+
+Tooltip.prototype = {
+  show: function(organism)
+  {
+    div = document.createElement("div");
+    $(div).empty();
+
+    div.style.left = organism.location.x + organism.radius;
+    div.style.top = organism.location.y + organism.radius;
+    div.setAttribute('class', 'tooltip');
+
+    var table = document.createElement("table");
+
+    table.appendChild(this.createRow('organism', "# " + organism.ID));
+    table.appendChild(this.createRow('generation', organism.generation));
+    table.appendChild(
+      this.createRow('velocity',
+        Stats.roundToDecimal(organism.velocity.mag(), 3)));
```

Comment:
sorry for the delay but 'velocity' and Stats.roundBlah are both params of this.createRow()

so they should start on the same column